docs/UserGuide/UserGuide.tex

```diff
@@ -35,5 +29,10 @@
 
 \maketitle
 
+\section{How to Use}
+\progname ~is an easy to use web application, you first sign-up with either an email and password, or through Google authentication. After logging into the website for the first time you should click "Preferences" and set those to let us know which flicks to recommend. You can also add friends by going to "Social" and then creating groups and inviting friends to join them. After going into a group the owner can start a voting session, where our algorithm will provide flicks based off of every user's preferences' in the group. However, just because one person disliked a genre does not mean that genre will not show up, it depends on everyone else's preferences.
```

Comment:
Fix quotes syntax for Social and Preferences. Also is "~" supposed to be present?